frontend/src/components/layout/Toast.tsx

```diff
@@ -0,0 +1,22 @@
+import { clearCookie, getCookie } from "../../functions/cookies";
```

Comment:
Since this ended up not being a component but simply a function that triggers the display of the toast, this file should probably be `/frontend/src/functions/makeToast.ts` instead of `Toast.tsx` :)